examples/advanced_agents/sandbox_agent/src/nat_sandbox_agent/sandbox/daytona_sandbox.py

```
@@ -0,0 +1,225 @@
+                stdout=result.stdout or "",
+                stderr=result.stderr or "",
+            )
+
+        except TimeoutError:
+            logger.warning(f"Command timed out after {timeout}s: {command[:50]}...")
+            return CommandResult(
+                exit_code=-1,
+                stdout="",
+                stderr=f"Command timed out after {timeout} seconds",
+            )
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)